### ISSUE TITLE
Dev Server starts but never render the project.

### ISSUE THREAD

**Issue by guilhermerera** [vercel/next.js#36427]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

    Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Pro
    Binaries:
      Node: 16.14.0
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant packages:
      next: 12.1.6-canary.6
      react: 18.0.0
      react-dom: 18.0.0

### Describe the Bug

My project was running on dev server. I closed the server to install `axios`, which happend with no issues. I runned the `npm run dev` command and the server never rendered, it is frozen on the "ready" message. I created a new folder, installed Next again, with the `npx create-next-app` command and copied all my project files into that folder and it was running with no issues, even with `axios` installed.
I took another pause and when I went back to the project, the same issue happend: the project is not rendering and is stuck on the "ready" message, as the image below shows:
![nextjs_bug](https://user-images.githubusercontent.com/85325067/165019073-6671fb9e-6ecf-4f65-a4a3-a65e05da9c18.png)

### Expected Behavior

The dev server should be able to render the project on localhost:3000

### To Reproduce

Start a project, worka little bit with it. Close the project, wait a little bit, get back to the same project and try to `npm run dev`.

**Comment by balazsorban44** [comment]
We will need a full reproduction to be able to debug. What do you mean by "close the project"?

**Comment by guilhermerera** [comment]
> We will need a full reproduction to be able to debug. What do you mean by "close the project"?
> 
> Please attach your code (preferably a simplified version that reproduces the problem) as well

Hello Sorban, thanks for your time and help. Sorry for not describing the issue in details.
By "close the project" I mean, literaly, closing the VSCode to do something else not related to programing.

You can find my project attached to this comment. It is a study project, so it is pretty simple with no sensitive information on it and it should be quick to download.

It will probably render with no problems the first few times but after a while  the `npm run dev` will get stuck at that point where I showed on the image above.

[NextJS_Error_report.zip](https://github.com/vercel/next.js/files/8554925/NextJS_Error_report.zip)

**Comment by balazsorban44** [comment]
So I'll assume you use the VSCode terminal to spin up the dev server. If you close the window without exiting the dev server, I'm not sure if VSCode properly finishes the process, which I guess might lead to something hanging in the background, but I'm not sure. :thinking:

**Comment by guilhermerera** [comment]
> So I'll assume you use the VSCode terminal to spin up the dev server. If you close the window without exiting the dev server, I'm not sure if VSCode properly finishes the process, which I guess might lead to something hanging in the background, but I'm not sure. 🤔

Yes, I do use the VSCode terminal. Thanks for your help!

For now, I will use the new folder/project workaround and keep that in mind the next time I'm working on the project to check if the error persists.  I might try to use the system terminal to run the server too.

**Comment by guilhermerera** [comment]
> So I'll assume you use the VSCode terminal to spin up the dev server. If you close the window without exiting the dev server, I'm not sure if VSCode properly finishes the process, which I guess might lead to something hanging in the background, but I'm not sure. 🤔

I found a "solution" for it. 
I managed to reproduce issue with a consistency: it was happening everytime I tried to `npm run dev` after I stoped the server (using `CTRL+C` or by closing the VSCode).

If I delete the `.next` folder on my project and `npm run dev` after that, the server runs correctly and it renders my project.
I'm not sure what causes the project to stop rendering, but I think it is inside the `.next` folder
